java/server/src/org/openqa/grid/web/Hub.java

```diff
@@ -19,6 +19,8 @@
 
 import com.google.common.collect.Maps;
 
+import com.sun.org.glassfish.gmbal.ManagedObject;
```

Comment:
And again. The reason it's bad is that if someone uses a JDK not produced by Oracle they won't have this class.